### PR TITLE
(fix) Move changelog job to it's own workflow

### PR DESCRIPTION
**Pull request by Barocena**
- moved the job to its own workflow file
- changed the label to `skip changelog`

### CODE REVIEW

File: .github/workflows/changelog.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    - uses: actions/checkout@v3
+    - name: Changelog check
+    
+      uses: Zomzog/changelog-checker@v1.3.0
+      with:
+        fileName: CHANGELOG.md 
+        noChangelogLabel: skip changelog
+        checkNotification: Simple
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (boatbomber) [inline .github/workflows/changelog.yml:21]: Add a trailing newline please, it helps future diffs and aligns with the formatting of our other workflows

**Review by boatbomber — CHANGES_REQUESTED** [review]

File: .github/workflows/changelog.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    branches:
+      - master
+jobs:
+  build:
+    name: Check Actions
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Changelog check
+
```
Comment (boatbomber) [inline .github/workflows/changelog.yml:14]: I think this empty line should go above the name not below it

**Review by boatbomber — APPROVED** [review]

File: .github/workflows/changelog.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+name: Changelog Check
```
Comment (Dekkonot) [inline .github/workflows/changelog.yml:1]:
```suggestion
name: Changelog Check

```

File: .github/workflows/changelog.yml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+on:
+  pull_request:
+    types: [assigned, opened, synchronize, reopened, labeled, unlabeled]
+    branches:
+      - master
```
Comment (Dekkonot) [inline .github/workflows/changelog.yml:6]:
```suggestion
      - master

```

**Review by Dekkonot — CHANGES_REQUESTED** [review]
Just for readability, can we add spacing like in the other workflow?

Sorry for the semantic requests. :-)

**Review by Dekkonot — APPROVED** [review]

**Pull request merged**